README.md

```diff
@@ -19,7 +19,7 @@ Exporting basics:
 Read the documentation or examples about the Prometheus Go client. In particular, you can check the [simple example]() which demonstrates usage of the `promhttp` - this includes a `.Handler()` function which returns an `http.Handler`. The Prometheus Go client exports many metrics by default (about the Go runtime, eg. garbage collection), so you can export just these default metrics by simply attaching the `promhttp` handler to an `http.Server`. For example, if you have a muxer in a variable called `mux`, you can call `mux.Handle("/metrics", promhttp.Handler())`. You should then be able to start the server, and see some default metrics being exported on `/metrics`.
```

Comment:
link here for simple example does not exist

Reply:
Assumes auto import could be worth rewording to once you have added this handle and imported the library you will see default...